src/api.rs

```diff
@@ -232,10 +230,27 @@ impl ThreadPool {
         unsafe {
             let job_a = JobImpl::new(op, LockLatch::new());
             self.registry.inject(&[job_a.as_job_ref()]);
-            job_a.latch.wait();
+            job_a.wait();
             job_a.into_result()
         }
     }
+
+    /// Sends `op` to the work queue to be eventually executed.
+    pub fn push_and_forget<'a, OP>(&'a self, op: OP)
```

Comment:
Indeed I don't like the name very much. Not sure what name I prefer, maybe `spawn`, `fork`, or `start`? I don't think we generally use terms like `push` in our API to refer to starting threads, do we?